.gitignore

```diff
@@ -3,6 +3,7 @@ node_modules/
 .nyc_output/
 coverage/
 dist/
+showcase/node_modules/
```

Comment:
whoops didn't mean to leave this in, i was flailing around trying to make atom not search showcase/nodemodules